fhir-demo-2024/src/pages/trifork.js

```diff
@@ -22,12 +23,62 @@ export default function Trifork({ location }) {
         </a>
       </h1>
       <Roles list={roles} />
-      <section className="work-in-progress">
-        <p>Participation confirmed, showcase content to be added.</p>
+      <section id="intro">
+        <p>Trifork offers both Platform-as-a-Service (PaaS) and consultancy.</p>
+        <h2>Platform</h2>
+        <ul>
+          <li><Link to="#thp">Trifork Health Platform</Link></li>
+        </ul>
+        <h2>Consultancy</h2>
+        <ul>
+          <li><Link to="#fut">Fælles Udbredelse af Telemedicin (FUT)</Link></li>
```

Comment:
Just a hint, I think fragment links within the same page work better with the regular `<a href="#fut">` syntax.